utils/SwiftIntTypes.py

```diff
@@ -90,16 +90,16 @@ def numeric_type_names_Macintosh_only():
 
 def all_integer_binary_operator_names():
     return ['<<', '>>', '&*', '&', '&+', '&-', '|', '^']
-    
```

Comment:
No purpose, but removing it creates churn that not everyone likes.  Its presence is not harmful.